### DISCUSSION TITLE
assets public ip with used proxy plugin

### DISCUSSION THREAD

**Discussion by rusmongoose** [glpi-project/glpi-agent#416] [General]
### Your question

I used proxy plugen with apache reverse proxy. in this mode 
public ip for all assets -  ip glpi-agent with proxy plugin.
How i can get public ip  all assets?

**Comment by g-bougard** [comment]
Hi @rusmongoose 
sorry I don't understand your question. What do you expect and where ?

**Comment by rusmongoose** [comment]
In the tab with the asset there is a line "public ip". When using a proxy, the ip of the proxy server gets there, and not the external ip of the asset.
If you use a direct connection to the glpi server, then this tab displays the external ip from which the asset is connected.
How can I get the external ip from which the asset connects using the proxy plugin?
![image](https://github.com/glpi-project/glpi-agent/assets/26715139/6d8739e6-1a0e-4529-b44c-45cdcfc0441d)

**Reply by g-bougard** [reply]
If you use the GLPI-Agent Proxy plugin, you'll actually always have the proxy agent ip. If you're using an Apache reverse proxy between computers and the proxy agent, maybe you can better directly redirect to GLPI. I'm not sure the proxy agent is useful behind a reverse proxy. But this is up to you.
